qdrant-landing/content/documentation/cloud/backups.md

```diff
@@ -1,36 +1,95 @@
 ---
 title: Backups
-weight: 30
+weight: 60
 ---
 
-# Backups
+# Cloud Backups
 
-There are situations where you need to restore your cluster because of application or system failure.
-In most cases you will have a source of truth for your data in a regular database and would be able to reindex the data into your Qdrant vector search cluster.
-However, encoding and uploading a big amount of data might require a long time.
-For high availability critical projects we highly recommend relying on replication, which is always a better option, because it guarantees the proper cluster functionality as long as at least one replica is running.
-For less critical use-cases you can make use of one of the available options.
+Qdrant organizes cloud instances as clusters. On occasion, you may need to
+restore your cluster because of application or system failure.
+
+You may already have a source of truth for your data in a regular database. If you
+have a problem, you could reindex the data into your Qdrant vector search cluster.
+However, this process can take time. For high availability critical projects we
+recommend replication. It guarantees the proper cluster functionality as long as
+at least one replica is running.
+
+For less critical use-cases you can set up automatic or self-service backups.
+
+## Prerequisites
+
+You can back up your Qdrant clusters though the Qdrant Cloud
+Dashboard at https://cloud.qdrant.io. This section assumes that you've already
+set up your cluster, as described in the following sections:
+
+- [Create a cluster](/documentation/cloud/create-cluster/)
+- Set up [Authentication](/documentation/cloud/authentication/)
+- Configure one or more [Collections](/documentation/concepts/collections/)
 
 ## Automatic backups
 
-The cloud platform offers an option for automatic backups of your clusters. It is possible to configure periodical file system level snapshots to restore a cluster from a hard copy.
-On the cluster settings section you can choose how often a backup should be taken and how many copies you want to keep.
+You can set up automatic backups of your clusters with our Cloud UI at
+https://cloud.qdrant.io. With the procedures listed in this page, you can set up
+snapshots on a daily/weekly/monthly basis. You can keep as many snapshots as you
+need. You can restore a cluster from the snapshot of your choice.
+
+> Note: Restoring a snapshot may create issues:
+> - The affected cluster is not available while a snapshot is being restored.
+> - If you changed the cluster setup after the copy was created, the new cluster 
```

Comment:
It's not a new cluster. The existing cluster will be reset.

Reply:
[Corrected](https://github.com/qdrant/landing_page/pull/585/files#diff-4682bb6a3b58fe427a5ab171f794a7499ba5d95088d01bcc2a9e819784c90ab6R39-R40)